public/gameInfo.php

```diff
@@ -560,36 +569,57 @@ function submitSetRequest(user, gameID) {
             $imageIngame = $gameData['ImageIngame'];
             $pageTitleAttr = attributeEscape($pageTitle);
 
+            function addMetadata($label, $gameDataValue, $gameHubs)
+            {
+                $output = '';
+
+                $first = true;
+                if ($gameHubs) {
+                    $hubPrefix = "[$label - ";
+                    foreach ($gameHubs as $hub) {
+                        if (substr($hub['Title'], 0, strlen($hubPrefix)) == $hubPrefix) {
+                            if ($first) {
+                                $first = false;
+                            } else {
+                                $output .= ", ";
```

Comment:
Minor thing - alternatively, I'd probably collect all the outputs in an array and implode it with a separator after the loop.
Then there's no need for the local `$first` variable and the `if/else` construct.
```
$hubLinks = [];
foreach ($gameHubs as $hub) {
    $hubLinks[] = "<a href=/game/" . $hub['gameIDAlt'] . ">$value</a>";
}
$output .= implode(', ', $hubLinks);
```

Not sure how it affects `$value == $gameDataValue` and the other check below - those might be reduced to filtering the array of hubs beforehand?

Reply:
Good idea. I forget about language-specific tricks like this.